dldashboard/web/build/asset-manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "files": {
     "main.css": "/static/css/main.f62cf940.chunk.css",
-    "main.js": "/static/js/main.68952394.chunk.js",
+    "main.js": "/static/js/main.638465d3.chunk.js",
```

Comment:
Are these build files artifacts of `yarn build`? Please also add how you generated them to the PR summary.

Reply:
Yes, it is generated through `yarn build`. I will add steps I followed to generate the PR